### PR TITLE
py-jupyterlab, py-jupyterlab_launcher: new ports

### PR DESCRIPTION
**Pull request by lpsinger**
@stromnov, would you like to maintain these, since you maintain most of the other jupyter ports?

### CODE REVIEW

**Comment by stromnov** [comment]
@lpsinger Yes, I can.

PS. Is it possible to change commit message to reflect actual new port name py-jupyterlab instead of py-jupyter?

File: python/py-jupyterlab/Portfile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-jupyterlab
+version             0.33.12
+revision            0
```
Comment (pmetzger) [inline python/py-jupyterlab/Portfile]: We probably don't need a revision 0 line.

File: python/py-jupyterlab/Portfile [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+categories-append   devel
+platforms           darwin
+license             BSD
+supported_archs     noarch
+
+python.versions     27 34 35 36
```
Comment (pmetzger) [inline python/py-jupyterlab/Portfile]: Can 3.7 be supported? Also I'm not sure new ports need to support 3.4 and 3.5.

File: python/py-jupyterlab/Portfile [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                    terminals, and custom components in a flexible, \
+                    integrated, and extensible manner.
+
+homepage            http://jupyter.org
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  4fe0074a1202cbc3833d1a85a3b8b9c5c97701c2 \
+                    sha256  04a14b38b218c87731b519ff0f7e133b28a613ad944617ac7e07480d6a36eb5f
```
Comment (pmetzger) [inline python/py-jupyterlab/Portfile]: These days we would traditionally also have a "size" line as part of "checksums".

File: python/py-jupyterlab_launcher/Portfile [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-jupyterlab_launcher
+version             0.11.2
+revision            0
```
Comment (pmetzger) [inline python/py-jupyterlab_launcher/Portfile]: We probably don't need the "revision 0".

File: python/py-jupyterlab_launcher/Portfile [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+categories-append   devel
+platforms           darwin
+license             BSD
+supported_archs     noarch
+
+python.versions     27 34 35 36
```
Comment (pmetzger) [inline python/py-jupyterlab_launcher/Portfile]: Can we support 3.7? And we probably don't need 3.4 and 3.5.

File: python/py-jupyterlab_launcher/Portfile [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+description         Launch an application built using JupyterLab.
+long_description    ${description}
+
+homepage            http://jupyter.org
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  1d6bb6fdabd30fd77ae4dafb154c357e519b262e \
+                    sha256  236a647f4c3f8417413643a918a893a5f662fb5d2fdccce2fd101e3cca2e7fd1
```
Comment (pmetzger) [inline python/py-jupyterlab_launcher/Portfile]: We now have a "size" line as part of "checksums".

**Comment by pmetzger** [comment]
@lpsinger If you could address the small fixes above, we can get your new ports merged...

**Comment by pmetzger** [comment]
@lpsinger ?

**Comment by pmetzger** [comment]
Merged. Thanks for the new ports, @lpsinger!

**Pull request merged**